test/e2e/manifest/testIPPool/ippool.yaml

```diff
@@ -3,7 +3,7 @@ kind: IPPool
 metadata:
   name: guestcluster-ippool-2
 spec:
-  type: public
+  type: Public
```

Comment:
please remove ip pool related test code and only add VM CRD for this PR. It's good to trace incremental change in one individual change, also follow agile methodology.

Reply:
ok